workers/ohsome_quality_analyst/indicators/currentness/indicator.py

```diff
@@ -82,103 +93,137 @@ def calculate(self) -> None:
-        else:
-            param_1 = 0.2
-        if years_since_last_edit <= 1:
-            param_2 = 1
-        elif years_since_last_edit <= 4:
-            param_2 = 0.6
-        else:
-            param_2 = 0.2
-        self.result.value = (param_1 + param_2) / 2
-        if median_diff == 0:
-            median_diff = "this year"
-        elif median_diff == 1:
-            median_diff = "in the last year"
+        self.result.value = int(self.result.timestamp_oqt.year) - int(self.median_year)
+        if self.result.value < 0.0:
+            self.result.value = 0.0
+
+        if last_edited_year != str(self.result.timestamp_oqt.year):
+            years_without_mapping = (
+                "Attention: There was no mapping"
+                " activity after {0} in this region.".format(last_edited_year)
+            )
         else:
-            median_diff = "in the last {0} years".format(median_diff)
+            years_without_mapping = ""
         self.result.description = Template(self.metadata.result_description).substitute(
-            years=median_diff,
+            years=self.result.value,
             layer_name=self.layer.name,
             end=self.end,
-            elements=self.contribution_sum,
+            elements=int(self.contribution_sum),
+            green=round(green_contributions_rel, 3),
+            yellow=round(yellow_contributions_rel, 3),
+            red=round(red_contributions_rel, 3),
+            median_years=self.result.value,
         )
 
-        if self.result.value >= self.threshold_yellow:
+        if self.result.value >= self.threshold_class_5:
             self.result.class_ = 5
             self.result.description = (
                 self.result.description + self.metadata.label_description["green"]
             )
-        elif self.result.value >= self.threshold_red:
+        elif self.result.value >= self.threshold_class_4:
+            self.result.class_ = 4
+            self.result.description = (
+                self.result.description + self.metadata.label_description["green"]
+            )
+        elif self.result.value >= self.threshold_class_3:
             self.result.class_ = 3
             self.result.description = (
                 self.result.description + self.metadata.label_description["yellow"]
             )
-        elif self.result.value < self.threshold_red:
+        elif self.result.value >= self.threshold_class_2:
+            self.result.class_ = 2
+            self.result.description = (
+                self.result.description + self.metadata.label_description["yellow"]
+            )
+        elif self.result.value < self.threshold_class_2:
             self.result.class_ = 1
             self.result.description = (
                 self.result.description + self.metadata.label_description["red"]
             )
         else:
             raise ValueError("Ratio has an unexpected value.")
+        self.result.description += years_without_mapping
```

Comment:
Do we need a new line here?

Reply:
Yes, otherwise the pre-commit throw an error

Reply:
I mean a new line in the description. Something like:
`        self.result.description += "\n" + years_without_mapping`